doc/api/timers.md

```diff
@@ -81,6 +81,17 @@ added: v11.0.0
 
 If true, the `Timeout` object will keep the Node.js event loop active.
 
+### `timeout.close()`
```

Comment:
Please move this before `hasRef`. We try to keep the documentation sorted alphabetically.